broadcast/transaction_options.go

```diff
@@ -31,15 +31,15 @@ type TransactionOpts struct {
 // and receive the transaction details and status.
 func WithCallback(callbackURL string, callbackToken ...string) TransactionOptFunc {
 	return func(o *TransactionOpts) {
-		o.CallbackToken = callbackURL
+		o.CallbackURL = callbackURL
 		if len(callbackToken) > 0 {
 			o.CallbackToken = callbackToken[0]
 		}
 	}
 }
 
-// WithMerkleProof will return merkle proof from Arc.
-func WithMerkleProof() TransactionOptFunc {
+// WithMerkleProofFromCallback will return merkle proof from Arc.
```

Comment:
I know that it's out of context, but this function won't `return merkle proof from Arc` as it is said in the comment but it's a functional options to enable it.

Reply:
You are right, fixed ✅